packages/skeleton-react/src/lib/components/Navigation/Navigation.tsx

```diff
@@ -157,6 +157,7 @@ export const NavTile: React.FC<NavTileProps> = ({
 	labelExpanded,
 	title,
 	selected = false,
+	type,
```

Comment:
@phamduylong I'm away from my workstation so I cannot test this directly right now. But you may need to set `type = "button"` so there's a default type present. This would be needed for the prop for both framework components.

If you can get that change, this should be good to merge!

Reply:
Donezo